### DISCUSSION TITLE
Copilot Vscode only doing single line completion

### DISCUSSION THREAD

**Discussion by DicksonWu654** [community/community#13389] [Copilot Conversations]
Issue Type: <b>Bug</b>

Hi there,

My Github Copilot only returns single line suggestions for anything, even the test case:

function calculateDaysBetweenDates(begin, end) {

From looking at other people's copilots it seems like their's are capable of doing multiline suggestions

VS Code version: Code 1.65.2 (Universal) (c722ca6c7eed3d7987c0d5c3df5c45f6b15e77d1, 2022-03-10T14:33:49.188Z)
OS version: Darwin arm64 21.3.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 3, 4|
|Memory (System)|8.00GB (0.09GB free)|
|Process Argv|--crash-reporter-id 8c5e02da-5db0-46bf-b816-b0446c03c30b|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
copilot|Git|1.9.5350
solidity|Jua|0.0.137
python|ms-|2022.2.1924087327
vscode-pylance|ms-|2022.3.2
jupyter|ms-|2022.2.1030672458
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6
vsliveshare|ms-|1.0.5449

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
pythonvspyl392:30443607
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
vsc1dst:30438360
pythonvs932:30410667
wslgetstarted:30449410
vsclayoutctrt:30451275
dsvsc009:30452663
pythonvsnew555:30457759
vscscmwlcmt:30438805
vscgsvid2:30447481
cppdebugcf:30451567
pynewfile477:30450038

```

</details>

**Comment by wicharek** [comment]
Hey, having the same issue :( Also VScode (Version: 1.68.1) on Mac. Were you able to figure this out?

**Comment by valentinbeggi** [comment]
Same issue here :(

**Comment by psstpakGithub** [comment]
Same for me

**Comment by davecheney** [comment]
@wicharek @valentinbeggi @psstpakGithub 

Please check your copilot settings, if you have enabled “block suggestions similar to public code” copilot may decline to provide a solution. 

If you change your setting, please take note of the instructions at the bottom of the page.

**Reply by psstpakGithub** [reply]
> @wicharek @valentinbeggi @psstpakGithub
> 
> Please check your copilot settings, if you have enabled “block suggestions similar to public code” copilot may decline to provide a solution.
> 
> If you change your setting, please take note of the instructions at the bottom of the page.

Suggestions matching public code is set to "Allow", but Copilot behaves differently on VS Code than in VS. In VS offers suggestions less frequently, and tends to offer single line code

**Reply by davecheney** [reply]
This sounds like a different issue, I suggest starting a new thread or contributing to this if appropriate https://github.com/github-community/community/discussions/19315#discussioncomment-3020928

**Reply by nishantgerald** [reply]
> This sounds like a different issue, I suggest starting a new thread or contributing to this if appropriate https://github.com/github-community/community/discussions/19315#discussioncomment-3020928

This discussion is deleted
